charts/hono/values.yaml

```diff
@@ -1708,7 +1711,7 @@ kafka:
       type: "pem"
       pemChainIncluded: true
       existingSecrets:
-      - "{{ .Release.Name }}-kafka-example-keys"
+      - "{{ include \"hono.fullname\" (dict \"dot\" .) }}-kafka-example-keys"
```

Comment:
As the above value is evaluated in the context of the Kafka chart, the `hono.fullname` template will produce `{ReleaseName}-kafka` here, meaning the complete value will be `{ReleaseName}-kafka-kafka-example-keys`.
Therefore, the additional `-kafka' part can be omitted above.

We could also add a comment there, to document the special logic:
```
      - "{{/* evaluated in kafka chart via tpl; kafka chart full name used here! */}}\
      {{ include \"hono.fullname\" (dict \"dot\" .) }}-example-keys"
```
(see also corresponding comment regarding `kafka-example-keys.yaml`)